packages/mdc-icon-button/constants.ts

```diff
@@ -22,7 +22,10 @@
  */
 
 export const cssClasses = {
-  ICON_BUTTON_ON: 'mdc-icon-button--on',
+  ICON: 'mdc-icon-button__icon',
+  ICON_BUTTON_ON: 'mdc-icon-button--on', // deprecated
```

Comment:
I like it marked it as a deprecated for backward compat. We can remove this later in major release.

Reply:
I tried to change code like this for notifying to developer...
```
/**
   * @deprecated
   * Use cssClasses.ON instead
   */
  get ICON_BUTTON_ON() {
    console.warn('MDCWeb: Use cssClasses.ON instead of cssClasses.ICON_BUTTON_ON');
    return 'mdc-icon-button--on';
  },
```
But console is blocked by tslint 😢